get_prism_data.R

```diff
@@ -54,6 +54,7 @@ download_prism=function(){
 #########################################################
 check_if_prism_files_present=function(prism_ls, years){
   #Extract out all the variable names and dates
+  if (length(prism_ls$files) == 0){return(FALSE)}
```

Comment:
can you add a comment like this?

#Check if prism directory is empty and no files are listed
if (length(prism_ls$files) == 0){return(FALSE)}
#If there are prism files present, extract out all the variable names and dates from their names
prism_ls = prism_ls %>%
.....